packages/runner-ct/src/.eslintrc.json

```diff
@@ -1,26 +1,5 @@
 {
   "extends": [
-    "plugin:@cypress/dev/react",
-    "plugin:@cypress/dev/tests"
-  ],
-  "parser": "@typescript-eslint/parser",
-  "settings": {
-    "react": {
-      "version": "16.12"
-    }
-  },
-  "rules": {
-    "arrow-body-style": "off",
-    "no-unused-vars": "off",
-    "react/jsx-filename-extension": [
-      "warn",
-      {
-        "extensions": [
-          ".js",
-          ".jsx",
-          ".tsx"
-        ]
-      }
-    ]
-  }
+    "../../reporter/src/.eslintrc.json"
+  ]
```

Comment:
Probably need a better place to have base eslintrc files... basically every package has copy-pasted the same one and made minor changes.